test-lib-openshift.sh

```diff
@@ -60,6 +62,13 @@ function ct_os_get_all_pods_status() {
   oc get pods -o custom-columns=NAME:.metadata.name,Ready:status.containerStatuses[0].ready
 }
 
+# ct_os_get_all_pods_name
+# --------------------
+# Returns the full name of all pods.
+function ct_os_get_all_pods_name() {
```

Comment:
Will this function be used separately?

IMPOV, more functions this file has, more hard to read it is...

Reply:
I can see it being used for when you want to get names of build/deploy pods, since those are by default ignored by `ct_os_get_pod_name`. But might be too much of a corner case. I am ok with removing the function.

Reply:
It makes sense. Thanks for explanation.